storage/inchi/src/test/java/org/openscience/cdk/inchi/InChIToStructureTest.java

```diff
@@ -232,4 +232,14 @@ public void s_penta_2_3_diene() throws Exception {
         assertThat(element.focus(), is(container.getAtom(4)));
         assertThat(element.winding(), is(ITetrahedralChirality.Stereo.CLOCKWISE));
     }
+
```

Comment:
Removed. New patches: https://github.com/cdk/cdk/compare/master...egonw:feature/substancedescriptor2

Plz comment on the NaN issue...